testing/machine/machine.go

```diff
@@ -64,6 +64,13 @@ func (m *Machine) CreateNamespace(namespace string) (TearDownFunc, error) {
 	return m.CreateLabeledNamespace(namespace, map[string]string{})
 }
 
+// DeleteNamespace deletes a namespace, it doesn't return an error if the namespace exists
```

Comment:
What does "it doesn't return an error if the namespace exists" mean?

Reply:
Done.